superset/migrations/versions/2023-05-11_12-41_4ea966691069_cross_filter_global_scoping.py

```diff
@@ -91,20 +94,24 @@ def downgrade():
                 chart_id = config.get("id")
                 if chart_id is None:
                     continue
-                scope = config.get("crossFilters", {}).get("scope", {})
+                scope = config.get("crossFilters", {}).get("scope", "")
                 new_chart_configuration[chart_id] = copy.deepcopy(config)
-                if scope == "global":
+                if scope.lower() == "global":
                     new_chart_configuration[chart_id]["crossFilters"]["scope"] = {
                         "rootPath": ["ROOT_ID"],
                         "excluded": [chart_id],
                     }
 
             json_metadata["chart_configuration"] = new_chart_configuration
-            del json_metadata["global_chart_configuration"]
+
+            if "global_chart_configuration" in json_metadata:
+                del json_metadata["global_chart_configuration"]
+
```

Comment:
nit: 
```suggestion
            json_metadata.pop('global_chart_configuration')
```

Reply:
`.pop()` fails if the key doesn't exist:

```python
>>> {}.pop(1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 1
```